nixos/modules/services/networking/iscsi/target.nix

```diff
@@ -0,0 +1,63 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.target;
+in
+{
+  ###### interface
+  options = {
+    services.target = with types; {
+      enable = mkOption {
+        type = bool;
+        default = false;
+        description = ''
+          Whether to enable the kernel's LIO iscsi target
+        '';
+      };
+
+      config = mkOption {
+        type = attrs;
+        default = {};
+        description = ''
+          Content of /etc/target/saveconfig.json
```

Comment:
There is no documentation for the contents of this file and the format is kind of strange. The parse is also handwritten in python. Not sure if that should be mentioned here, but this description isn't too helpful.

Reply:
How did you come up with the data in the test?

Reply:
I used targetcli and cut as much away from the config it generates as I could, without it breaking.